quipucords/scanner/input_log.py

```diff
@@ -0,0 +1,301 @@
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+
+"""Log all input to the filesystem for later debugging."""
```

Comment:
Can we have a better name than `input_log` I don't think it is very descriptive. What input? Its really more a recorder or logger.

Reply:
How about `remote_data_log`? I'm not tied to either name., I don't think they're that good, but I've been trying to think of a better name and I'm drawing a blank right now.

Reply:
Or maybe `scan_data_log`.